cellengine/resources/gate.py

```diff
@@ -193,81 +152,345 @@ def create(
         x2: float,
         y1: float,
         y2: float,
-        label: List[str] = [],
-        gid: str = None,
+        label: List[float] = [],
+        gid: Optional[str] = None,
         locked: bool = False,
-        parent_population_id: str = None,
-        parent_population: str = None,
         tailored_per_file: bool = False,
-        fcs_file_id: str = None,
-        fcs_file: str = None,
+        fcs_file_id: Optional[str] = None,
+        fcs_file: Optional[str] = None,
         create_population: bool = True,
-    ) -> RectangleGate:
-        g = format_rectangle_gate(**get_args_as_kwargs(cls, locals()))
-        return cls(g)
+        parent_population_id: str = None,
+    ) -> Union[RectangleGate, Tuple[RectangleGate, Population]]:
+        """Creates a rectangle gate for posting to the CellEngine API.
 
+        Args:
+            experiment_id (str): The ID of the experiment.
+            x_channel (float): The name of the x channel to which the gate applies.
+            y_channel (float): The name of the y channel to which the gate applies.
+            name (str): The name of the gate x1 (float): The first x coordinate
+                (after the channel's scale has been applied).
+            x2 (float): The second x coordinate (after the channel's scale
+                has been applied).
+            y1 (float): The first y coordinate (after the channel's scale has
+                been applied).
+            y2 (float, optional): The second y coordinate (after the channel's scale has
+                been applied).
+            label (float, optional): Position of the label. Defaults to the
```

Comment:
```suggestion
            label (List[float], optional): Position of the label. Defaults to the
```

Reply:
Still pending.

Reply:
Oh, this change was made to the wrong arg (`y2` instead of `label`).

Reply:
oops, fixed